rules/opentitan_test.bzl

```diff
@@ -200,14 +216,15 @@ def cw310_params(
         "exclusive",
     ]
     kwargs.update(
-        args = required_args + args,
+        args = default_args + args,
```

Comment:
Just a nit (after the fact) "default_args" makes it seem like setting args will override it, while required args could be reasonable expected to be prepended unconditionally. I'd expect to see args assigned to default_args if it's undefined.

Thanks for getting the args through bazel to the test runner appropriately, it was frustrating not being able to use --test_args because OTT wanted args in a specific sequence.

Reply:
That's not how this rule works, though. Instead, earlier args are overridden by subsequent assignment to the same options, so "default_args" are overridden by "args", and those can be overridden by "--test_arg" options to bazel.

The requirement for opentitan_functest is that the test runner operates on args in this way--Otherwise, there would need to be a different rule to accommodate that different class of runner.